packages/labs/analyzer/test-files/js/functions/functions.js

```diff
@@ -36,3 +36,46 @@ export function function2(a, b = false, ...c) {
 export default function () {
   return 'default';
 }
+
+/**
+ * This function has an overloaded signature in TS.
```

Comment:
What does that mean exactly? Is this file associated with `ts/functions/src/functions.ts` so that those overloads apply here? I would have thought that these were separate modules.

Reply:
Yes, the same tests are run twice (once for each analysis result from files in the `ts` and `js` directories) so the doc comments that those tests check for need to match. For overloaded functions in particular, the comments for the single declaration in JS need to match those from the particular overload that they're extracted from in TS. These could be rewritten to make what's happening here clearer though.

However, it also sounds like I need to update this PR to handle overloads by producing multiple entries rather than trying to merge into a single one and I'm not sure how this is going to affect what's in the JS version of these fixtures yet. I'll try to make these more self-explanatory once I know what's happening with them.